src/api/driver.ts

```diff
@@ -10,6 +10,8 @@ export const getRegionDriver = async (
     true
   );
 
+export const getDriver = async () => await GET(`/driver/search`, true);
```

Comment:
랜딩 페이지 드라이버 프로필은 **모든 유저**가 조회 가능해야 할 것 같아요.
즉, 로그인 여부 및 JWT 유효성 여부와 상관없이 항상 조회가 가능해야 합니다.
확인해보니, 백엔드에서 JWT가 없어도 조회가 가능하도록 api를 만들어주신 것 같아요.
그럼 해당 api를 호출할 때, JWT를 포함하지 않아도 되기 때문에 두번째 파라미터를 `false`로 바꿔주시면 좋을 것 같아요.
두번째 파라미터는 api 호출시 JWT를 함께 보낼지에 대한 여부입니다!
```suggestion
export const getDriver = async () => await GET(`/driver/search`, false);
```